src/content/docs/apm/agents/ruby-agent/api-guides/ruby-custom-instrumentation.mdx

```diff
@@ -237,4 +237,17 @@ class MyEventWatcher
 end
 ```
 
-For more information, see [Tracer#start_transaction_or_segment in the New Relic RubyDoc](https://www.rubydoc.info/github/newrelic/newrelic-ruby-agent/NewRelic/Agent/Tracer#start_transaction_or_segment-class_method).
+For more information, see [Tracer#start_transaction_or_segment in the New Relic RubyDoc](https://www.rubydoc.info/gems/newrelic_rpm/NewRelic/Agent/Tracer#start_transaction_or_segment-class_method).
+
+### Instrumenting Threads
+
+To support applications that want some threads instrumented by New Relic, a class is available in 8.7.0 and above, `NewRelic::TracedThread`, that will create a thread that includes New Relic instrumentation. 
+Use this class as a replacement for the native Thread class. Please see our [API documentation](https://www.rubydoc.info/gems/newrelic_rpm/NewRelic) for more details. 
```

Comment:
I wonder if we can hack this link in advance? I think it'll end up being /NewRelic/TracedThread